src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -107,6 +109,30 @@
  *     }
  * </pre>
  * <p>
+ * To configure the check for whitespace only after LITERAL_TRY tokens:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;WhitespaceAfter&quot;&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_TRY&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class TryStatement {
+ *     public static void main(String[] args) throws IOException {
+ *       try(InputStream ignored = System.in) { // violation ''try' is not followed by whitespace'
```

Comment:
Put this example to default config example.
As we add this token to default tokens.
No need for extra example section.

Reply:
Changes Done